runtime/dancebox/src/lib.rs

```diff
@@ -1530,13 +1530,20 @@ impl pallet_stream_payment::TimeProvider<TimeUnit, Balance> for TimeProvider {
 
 type StreamId = u64;
 
+parameter_types! {
+    pub const OpenStreamHoldAmount: Balance = 100 * UNIT;
```

Comment:
This is way to much haha.

Reply:
We probably should calculate how much size it takes to hold a deposit in storage. I think there is something similar done in pallet-assets

Reply:
You mean [`currency::deposit`](https://github.com/moondance-labs/tanssi/blob/master/runtime/dancebox/src/lib.rs#L160)? Like it's used for [pallet_identity](https://github.com/moondance-labs/tanssi/blob/master/runtime/dancebox/src/lib.rs#L1546)